src/main/objects/commons/Consts.js

```diff
@@ -250,8 +250,8 @@ var keyFromPitchBasenoteAndMode = {
     "70,5,minor": 'a#',
     "70,6,minor": 'bb',
     "71,6,minor": 'b',
+    "69,6,major": 'Bbb'
```

Comment:
Tutaj warto dodać już wszystkie możliwości, ale może utwórzmy osobny ticket na to, bo raczej na razie będzie działało bez tych wartości.